README.md

```diff
@@ -58,20 +58,20 @@ Layers:
 
 # ogr_fdw_info -s /tmp/test -l pt_two
 
-CREATE SERVER myserver
+CREATE SERVER "myserver"
   FOREIGN DATA WRAPPER ogr_fdw
   OPTIONS (
     datasource '/tmp/test',
     format 'ESRI Shapefile' );
 
-CREATE FOREIGN TABLE pt_two (
+CREATE FOREIGN TABLE "pt_two" (
   fid integer,
-  geom geometry(Point, 4326),
-  name varchar,
-  age integer,
-  height real,
-  birthdate date )
-  SERVER myserver
+  "geom" geometry(Point, 4326),
```

Comment:
As I mentioned in the other PR, I don't want people to be required to quote column names, and I'd rather not quote them when I do not have to. There's various options in the `IMPORT FROM SCHEMA` around twiddling with how much name laundering happens, and my default is "launder the shit out of them". I'm not opposed to a "literal column name" option, but I don't want it as the default.

Reply:
As I think I mentioned the problem is not about literal column names and very happy with the "launder the shit out of them" approach. The issue is with **reserved words** so, for example, if an ogr source has the column name "current_date" of type date then the current code will output "current_date integer" and boom.

I've added the current list of reserved words for the development version of postgresql (13) and now only quote when the column name is in the reserved word list, but I can make quoting optional if you prefer.